docs/Rapid/3-User Manual/1-rapid-overview/1-rapid-overview.md

```diff
@@ -1,3 +1,51 @@
+
+Sounds great, but how do we do that?
+
+## Major Features
+
+The Rapid Platform can be broken down into several main concepts
+
+### Your Site
+
+![Rapid login screen](Rapid_Login_Screen.png)
+
+Rapid is used on the web. When first setting it up you will have a dedicated place to store everything we call a Site. Your Site can interact with other places on the web that help solve particular activities, which we call Experiences.
+
+Ultimately, no matter how you interact with Rapid everything gets stored in one place; one source of truth, your Site.
+
+You can learn more about accessing your site and authentication in our [accessing your site](/docs/Rapid/1-Getting%20Started/accessing-your-site/accessing-your-site.md) page. 
+
+### Explorer
+
+![Explorer Screen](Explorer_Example.png)
+
+<!-- TODO: Update files and tasks link here, we don't have a place that talks about files or tasks as a concept yet -->
+
+Explorer is a looking glass into your site data. This experiences goal is to help you manage data on your site.
+If you're looking to [track information](/docs/Rapid/3-User%20Manual/2-Explorer/2-Items/1-items-overview/1-items-overview.md), organise [files](/docs/Rapid/1-Getting%20Started/1-welcome/1-welcome.md), manage [tasks](/docs/Rapid/3-User%20Manual/2-Explorer/0-navigating-explorer/0-navigating-explorer.md) or do anything else with your data, Explorer is key.
+
+### Workflow
+
+![Workflow Screen](Workflow_Example.png)
+
+Based off the popular [BPMN 2.0](https://camunda.com/bpmn/), [Workflow](/docs/Rapid/3-User%20Manual/4-Workflow/workflow-introduction.md) is a process diagramming and automation tool. Its rare for work to be done in a vacuum, Workflow helps you describe the flow of labour throughout your business. This helps self document what should happen when, and why. With a good description of what happens, Workflow can also <a href="https://docs.rapidplatform.com/training/Using%20Workflow">start automating</a> the creation of tasks and flow of information to help you and your team focus on getting things done.
+
+### Tasks and the Task Experience
+
+![Tasks Screen](Tasks_Example.png)
+
+Tasks are a fundamental unit of the Rapid Platform. A task describes a singular activity in time owned by someone.
+With a good set of processes under your belt, tasks are an excellent way to focus on what needs to be done and minimise the amount of chasing up information you need to do.
+
+<!-- TODO: Write up tasks experience overview and update link -->
+
+When you're in the groove having a well organised list of tasks can really help, that's what the [Tasks Experience](/docs/Rapid/3-User%20Manual/2-Explorer/0-navigating-explorer/0-navigating-explorer.md) was built for.
```

Comment:
When you're in the groove, having a well-organised list of tasks can really help. That's what the [Tasks Experience](/docs/Rapid/3-User Manual/2-Explorer/8-Tasks/8-Tasks.md) was built for.